app/app/static/custom/ourscript.js

```diff
@@ -0,0 +1,117 @@
+var timers = [];
+var timerIntervals = [];
+
+function showAddToQueue() {
+	$('#addToQueue').css('display', 'block');
+}
+
+function hideAddToQueue() {
```

Comment:
I presume you want to reset the form whenever this function is executed?, then hides `addToQueue`. Can you add a comment that says in what scenario I would like to use this function?

eg.

```
This function is used whenever ...
```